content/hi/docs/tasks/configmap-secret/managing-secret-using-kubectl.md

```diff
@@ -0,0 +1,187 @@
+---
+title: kubectl का उपयोग करके सीक्रेट्स का प्रबंधन
+content_type: task
+weight: 10
+description: kubectl कमांड लाइन का उपयोग करके सीक्रेट ऑब्जेक्ट बनाना।
+---
+
+<!-- overview -->
+
+यह पेज आपको `kubectl` कमांड-लाइन टूल का उपयोग करके कुबेरनेट्स {{<glossary_tooltip text="सीक्रेट्स" term_id="secret">}} को कैसे बनाने, संपादित करने, प्रबंधित करने और हटाने का तरीका दिखाता है।
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+<!-- steps -->
+
+## सीक्रेट बनाएं
+
+एक `Secret` ऑब्जेक्ट संवेदनशील डेटा जैसे क्रेडेंशियल्स को स्टोर करता है जिसका उपयोग पॉड्स द्वारा सेवाओं तक पहुंचने के लिए किया जाता है। उदाहरण के लिए, आपको डेटाबेस तक पहुंचने के लिए आवश्यक उपयोगकर्ता नाम और पासवर्ड को स्टोर करने के लिए एक सीक्रेट की आवश्यकता हो सकती है।
+
+आप कमांड में रॉ डेटा पास करके या क्रेडेंशियल्स को फाइलों में स्टोर करके सीक्रेट बना सकते हैं जिन्हें आप कमांड में पास करते हैं। निम्नलिखित कमांड एक सीक्रेट बनाते हैं जो उपयोगकर्ता नाम `admin` और पासवर्ड `S!B\*d$zDsb=` को स्टोर करता है।
+
+### रॉ डेटा का उपयोग करें
+
+निम्नलिखित कमांड चलाएं:
+
+```shell
+kubectl create secret generic db-user-pass \
+    --from-literal=username=admin \
+    --from-literal=password='S!B\*d$zDsb='
+```
+आपको अपनी स्ट्रिंग्स में विशेष वर्णों जैसे `$`, `\`, `*`, `=`, और `!` को एस्केप करने के लिए एकल उद्धरण `''` का उपयोग करना होगा। यदि आप ऐसा नहीं करते हैं, तो आपका शेल इन वर्णों की व्याख्या करेगा।
```

Comment:
Can we translate escape here?

Reply:
@divya-mohan0209  Corrected the escape character explanation